packages/experiments/src/components/Toggle/Toggle.state.ts

```diff
@@ -18,10 +20,17 @@ export class ToggleState extends BaseState<IToggleProps, IToggleViewProps, ITogg
       checked: !!props.defaultChecked,
       text: !!props.defaultChecked ? props.onText : props.offText,
       onChange: this._noop,
-      onClick: this._onClick
+      onClick: this._onClick,
+      toggleButton: this._toggleButton
```

Comment:
If you're gonna have a ref as a prop, you should at least call it a `toggleButtonRef`